app/Main.hs

```diff
@@ -20,6 +23,9 @@ defaultSettings = Settings { bindAddress = "0.0.0.0"
                            , bufferSize = 2^11
                            , authentication = ""
                            , realm = ""
+                           , https = False
```

Comment:
I was thinking about having HTTP and HTTPS at the same time using a single process.

What do you think about this?
```haskell
http = Just "8080" | Nothing
https = Just "8081" | Nothing
```